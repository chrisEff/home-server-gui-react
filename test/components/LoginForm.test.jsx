const React = require('react')
const LoginForm = require('../../src/components/LoginForm')

const { shallow } = require('enzyme')

describe('LoginForm', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<LoginForm />)
		expect(wrapper).toMatchSnapshot()
	})
})
