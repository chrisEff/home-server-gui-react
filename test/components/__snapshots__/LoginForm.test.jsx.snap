// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should render correctly 1`] = `
<div
  className="loginForm_12to336"
>
  <br />
  <input
    className="input_1bqe76u"
    name="apiUrl"
    placeholder="API URL"
    type="text"
  />
  <br />
  <input
    className="input_1bqe76u"
    name="apiUser"
    placeholder="API User"
    type="text"
  />
  <br />
  <input
    className="input_1bqe76u"
    name="apiKey"
    placeholder="API Key"
    type="password"
  />
  <br />
  <div
    className="button_1f1tgwh"
    onClick={[Function]}
  >
    OK
  </div>
</div>
`;
